controller/podcontroller.go

```diff
@@ -239,6 +229,11 @@ func (c *PodController) checkReadyStatus(oldpod, newpod *apiv1.Pod, cluster *crv
 						found, err := kubeapi.Getpgtask(c.PodClient, &tmptask, clusterName+"-stanza-create", newpod.ObjectMeta.Namespace)
 						if !found && err != nil {
 							backrestoperator.StanzaCreate(newpod.ObjectMeta.Namespace, clusterName, c.PodClientset, c.PodClient)
+							
+							// Disable autofailover if the autofail label is set to "false" on the pgcluster
+							if cluster.ObjectMeta.Labels[config.LABEL_AUTOFAIL] == "false" {
+								 util.ToggleAutoFailover(c.PodClientset, false, cluster.Name, cluster.ObjectMeta.Namespace)
+							}								 
```

Comment:
Would this be 'cleaner'? It sounds like the intent it to ensure that the toggle is equivalent to the value of the label. So, why test for it and not just set it to the value? If the label is anything other than ```1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False``` then it will just move on.

```
autofail, err := strconv.ParseBool(cluster.ObjectMeta.Labels[config.LABEL_AUTOFAIL])
if err != nil {
    util.ToggleAutoFailover(c.PodClientset, autofail, cluster.Name, cluster.ObjectMeta.Namespace)
}
```

https://golang.org/pkg/strconv/#ParseBool